File: iCovieApi/iCovieApi/Models/Master/PropertyTypeModel.cs
﻿using System;

using System;
using Newtonsoft.Json;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace iCovieApi.Models
{
    public class PropertyTypeModel
    {
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int id { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string propertytype { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public string shortname { get; set; }


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int cid { get; set; }
    }
}

File: iCovieAdmin-Ng/src/app/core/auth/_models/Masters/propertytype.model.ts


export class PropertyTypeModel {
    id: number;    
    propertytype:string;   
    cid:number; 
    isCore: boolean = false;    
    clear(): void {
        this.id= undefined;
        this.propertytype= '';        
        this.cid = undefined;      
        this.isCore=false;
	}
}

